distribution/lib/Standard/Base/0.0.0-dev/src/Data/Pair.enso

```diff
@@ -248,6 +250,17 @@ type Pair
         f self.second
         Nothing
 
+    ## PRIVATE
+       Convert to a display representation of this Pair.
+    to_display_text : Text
+    to_display_text self =
+        first = self.first.to_display_text
+        second = self.second.to_display_text
+        if first.length + second.length < 75 then "Pair(" + first + ", " + second + ")" else
+            first_trim = if first.length > 38 then first.take 37 + " …" else first
+            second_trim = if second.length > 38 then second.take 37 + " …" else second
```

Comment:
While 37 + 37 is 74, together with the three dots added there the whole messages is going to exceed the 80 characters limit.